src/main/scala/org/zalando/spearheads/innkeeper/services/EskipRouteService.scala

```diff
@@ -0,0 +1,64 @@
+package org.zalando.spearheads.innkeeper.services
+
+import java.time.LocalDateTime
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import com.google.inject.Inject
+import org.zalando.spearheads.innkeeper.api.JsonProtocols._
+import org.zalando.spearheads.innkeeper.api.{EskipRoute, EskipRouteWrapper, NewRoute, RouteName}
+import org.zalando.spearheads.innkeeper.dao.{PathRow, RouteRow, RoutesRepo}
+import spray.json.pimpString
+
+class EskipRouteService @Inject()(routesRepo: RoutesRepo, routeToEskipTransformer: RouteToEskipTransformer) {
+
+  private val -> = "\n -> "
```

Comment:
I would inline it. I don't think the eskip format will change in order to have this extracted. It would also be nicer to read in the string interpolation. I would put the new line in the call to mkString.